spec/lib/embed/viewer/media_spec.rb

```diff
@@ -17,6 +19,7 @@
   end
 
   describe '#body_html' do
+    before { stub_purl_response_with_fixture(video_purl) }
     it 'includes some dummy content' do
```

Comment:
This test is ensuring that the `MediaTag` is being implemented correctly.  Since the `MediaTag` class is responsible for generating the markup w/i the `video`/`audio` tags, the tests are in that class: https://github.com/sul-dlss/sul-embed/blob/streaming-server-config/spec/lib/embed/media_tag_spec.rb

Perhaps what would be more useful is a change in the test description (although I didn't actually change the test functionally, just stubbed the PURL object since we're accessing it from the `MediaTag` class now.